### DISCUSSION TITLE
Group field

### DISCUSSION THREAD

**Discussion by forzaxhx** [wazuh/wazuh#14130] [General]
The 'group.keyword' field of the 'wazuh-monitoring' Elasticseach index should be a field of the 'wauz-alerts' index. With this, the user could filter by group, and dont need to exclude agents manually

**Comment by mdiego92** [comment]
Hello forzaxhx,

There is a workaround that could be useful for you in this case. You could set in the group configuration a label:

```
<agent_config>
       <labels>
               <label key="group">testgroup</label>
       </labels>
</agent_config>
```

After that, you will need to refresh the index. Please go to the main menu > **Management** > **Stack Management**. Then select **Index Patterns** > **wazuh-alerts-*** > and the **refresh** button.
![image](https://user-images.githubusercontent.com/62477966/177804735-4ea7fa8c-ce4b-4064-9c66-49c031405990.png)

You could select the filter `agent.labels.group` to filter by the group in the security events section:

![image](https://user-images.githubusercontent.com/62477966/177805049-d03949a6-e297-46ba-a0cf-69d4babeed2d.png)

Please let us know if this could work for you.

Thank you very much.

**Reply by forzaxhx** [reply]
Thanks, it works for me 👍
